scene/gui/line_edit.cpp

```diff
@@ -822,7 +822,7 @@ void LineEdit::_notification(int p_what) {
 				case HORIZONTAL_ALIGNMENT_FILL:
 				case HORIZONTAL_ALIGNMENT_LEFT: {
 					if (rtl) {
-						x_ofs = MAX(style->get_margin(SIDE_LEFT), int(size.width - style->get_margin(SIDE_RIGHT) - (text_width)));
+						x_ofs = MAX(style->get_margin(SIDE_LEFT), int(size.width - ceil(style->get_margin(SIDE_RIGHT) + (text_width))));
```

Comment:
This should use our own `Math::ceil()` method for consistency (from `core/math/math_funcs.h`, might already be implicitly included).